### PR TITLE
Options for contact: mail and Slack

### PR DESCRIPTION
**Pull request by iMylene**

### CODE REVIEW

File: content/contact.md [hunk 1]
```diff
@@ -8,9 +8,9 @@ weight: 5
 
 The BAPC 2021 is hosted by [Vrije Universiteit Amsterdam](https://www.vu.nl/), and organized by a committee of [study association STORM](https://storm.vu). If you have any questions or want to contact the BAPC 2021 committee, these are the ways to do it:
 
-***Mail*** - You can drop us an email at: *organization “at” BAPC “dot” EU*.
+***Mail*** — You can drop us an email at: *organization “at” BAPC “dot” EU*.
 
-***Slack*** - The community created a [BAPC Slack](https://benelux-nwerc.slack.com/). Feel free to reach out to us on the *#general* channel in the Slack. For access, visit [this link](https://bapc-slack-enroll.herokuapp.com) with token `I heart Benelux`.
```
Comment (mvr320) [inline content/contact.md]:
```suggestion
***Slack*** - The BAPC community maintains a [Slack](https://benelux-nwerc.slack.com/). Feel free to reach out to us on the *#general* channel in the Slack. For access, visit [this link](https://bapc-slack-enroll.herokuapp.com) with token `I heart Benelux`. Questions on how to organize a preliminary, help with the ICPC system or general help can be asked here.
```

File: content/contact.md [hunk 2]
```diff
@@ -6,7 +6,11 @@ weight: 5
 
 # Contact
 
-If you have any questions or want to the contact the BAPC 2021 committee, please drop us an email at: organization “at” BAPC “dot” EU.
+The BAPC 2021 is hosted by [Vrije Universiteit Amsterdam](https://www.vu.nl/), and organized by a committee of [study association STORM](https://storm.vu). If you have any questions or want to contact the BAPC 2021 committee, these are the ways to do it:
+
+***Mail*** — You can drop us an email at: *organization “at” BAPC “dot” EU*.
+
+***Slack*** — The BAPC community maintains a [Slack](https://benelux-nwerc.slack.com/). Feel free to reach out to us on the *#general* channel in the Slack. For access, visit [this link](https://bapc-slack-enroll.herokuapp.com) with token `I heart Benelux`. Questions on how to organize a preliminary, help with the ICPC system or general help can be asked here.
```
Comment (mvr320) [inline content/contact.md:13]:
```suggestion
***Slack*** — The BAPC community maintains a [Slack](https://benelux-nwerc.slack.com/). Feel free to reach out to us on the *#general* channel in the Slack. For access, visit [this link](https://bapc-slack-enroll.herokuapp.com) with token `I heart Benelux`. Questions on how to organize a preliminary, help with the ICPC system or general help may be asked here.
```

File: content/contact.md [hunk 3]
```diff
@@ -6,7 +6,11 @@ weight: 5
 
 # Contact
 
-If you have any questions or want to the contact the BAPC 2021 committee, please drop us an email at: organization “at” BAPC “dot” EU.
+The BAPC 2021 is hosted by [Vrije Universiteit Amsterdam](https://www.vu.nl/), and organized by a committee of [study association STORM](https://storm.vu). If you have any questions or want to contact the BAPC 2021 committee, these are the ways to do it:
+
+***Mail*** — You can drop us an email at: *organization “at” BAPC “dot” EU*.
```
Comment (mvr320) [inline content/contact.md:11]: Move this one down?

And than tell people this is the less preferred alternative?

**Review by mvr320 — CHANGES_REQUESTED** [review]
Move the mail one down?

And than tell people this is the less preferred alternative?

**Review by mvr320 — COMMENTED** [review]
Looks fine, if someone else also agrees this can be merged!

**Review by EBaalhuis — COMMENTED** [review]
looks good

**Review by mvr320 — APPROVED** [review]

**Pull request merged**